src/utilities/filetypes/test/WorkflowJSON_GTest.cpp

```diff
@@ -1170,3 +1170,9 @@ TEST(Filetypes, RunOptions_ForwardTranslate_BackwardCompatibility) {
   EXPECT_EQ("my_ruby_file.rb", workflow2->runOptions()->customOutputAdapter()->customFileName());
   EXPECT_EQ("MyOutputAdapter", workflow2->runOptions()->customOutputAdapter()->className());
 }
+
+TEST(Filetypes, RunOptions_NoFtOptions) {
+  // Test for #4815
+  const RunOptions options;
+  EXPECT_EQ("", options.forwardTranslateOptions());
+}
```

Comment:
Before Fix, this returns `"null"` so it would throw on https://github.com/NREL/OpenStudio-workflow-gem/blob/cf99e6096f33b717df79a71c245a4d7aa4ccb7a6/lib/openstudio/workflow/adapters/input/local.rb#L368

`Error: undefined method `each' for nil:NilClass in :/ruby/2.7.0/gems/openstudio-workflow-2.3.1/lib/openstudio/workflow/adapters/input/local.rb:368:in `ft_options'`